packages/material-ui-lab/src/SpeedDial/SpeedDial.js

```diff
@@ -117,7 +105,8 @@ class SpeedDial extends React.Component {
       const nextAction = clamp(focusedAction + actionStep, 0, this.actions.length - 1);
       const nextActionRef = this.actions[nextAction];
       nextActionRef.focus();
-      this.setState({ focusedAction: nextAction, nextItemArrowKey });
```

Comment:
I was hoping to improve the performance of the component with this change, it doesn't help much.